modules/nft/keeper/nft.go

```diff
@@ -38,7 +38,8 @@ func (k Keeper) GetNFTs(ctx sdk.Context, denom string) (nfts []exported.NFT) {
 	return nfts
 }
 
-//Authorize check if the sender is the issuer of nft, if it returns nft, if not, return an error
+// Authorize checks if the sender is the issuer of the given NFT
```

Comment:
```suggestion
// Authorize checks if the sender is the owner of the given NFT
```

owner can be transferred to another